Library/Formula/asio.rb

```diff
@@ -0,0 +1,61 @@
+class Asio < Formula
+  desc "Cross-platform C++ Library for asynchronous programming"
+  homepage "http://think-async.com/Asio"
+  url "https://downloads.sourceforge.net/project/asio/asio/1.10.6%20%28Stable%29/asio-1.10.6.tar.bz2"
+  sha256 "e0d71c40a7b1f6c1334008fb279e7361b32a063e020efd21e40d9d8ff037195e"
+  head "https://github.com/chriskohlhoff/asio.git"
+
+  devel do
+    url "https://downloads.sourceforge.net/project/asio/asio/1.11.0%20%28Development%29/asio-1.11.0.tar.bz2"
+    sha256 "4f7e13260eea67412202638ec111cb5014f44bdebe96103279c60236874daa50"
+  end
+
+  option "with-boost-coroutine", "use Boost.Coroutine to implement stackful coroutines"
```

Comment:
Start the option explanation with a capital letter please, `"Use Boost.Coroutine to implement stackful coroutines"` in this case.